src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -288,6 +288,17 @@ hmi_apis::Common_Language::eType MessageHelper::CommonLanguageFromString(
   return hmi_apis::Common_Language::INVALID_ENUM;
 }
 
+hmi_apis::FunctionID::eType MessageHelper::HMIFunctionIDFromString(
```

Comment:
@LitvinenkoIra do we have Message Helper test to add test to this new function?

Reply:
@LuxoftAKutsan done 2d523133b33badd56fa1c4b17332a3523d4fb905